Andrei_Bakaushyn/06. Ansible_Workshop/nginx_setup.yml

```diff
@@ -0,0 +1,91 @@
+---
+- name: Install and Configure Nginx with Virtual Hosts and Check Availability
+  hosts: all
+  become: yes
+
+  vars:
+    vhost1_domain: vhost1.example.com
+    vhost2_domain: vhost2.example.com
+
+  tasks:
+    - name: Install Nginx
+      apt:
+        name: nginx
+        state: latest
+        update_cache: yes
+
+    - name: Start Nginx and Enable at Boot
+      service:
+        name: nginx
+        state: started
+        enabled: yes
+
+    - name: Create directories for VirtualHosts
+      file:
+        path: "{{ item }}"
+        state: directory
+        mode: '0755'
+      loop:
+        - /var/www/vhost1
```

Comment:
please use it as variable in array of virtual hosts
 example 
```yaml
        virtual_hosts:
          - { name: "vhost1", root_folder: "/var/www/vhost1", dns: "..."}
          - {  name: "vhost2", root_folder: "/var/www/vhost2", dns: "..."}
```
and call them in template